sbin/overc-cctl

```diff
@@ -575,9 +585,9 @@ function add_container {
 	if [ ${btrfs} == 1 ]; then
 		btrfs subvolume create ${pathtocontainer}
 	else
-		mkdir -p ${pathtocontainer}
+		mkdir -p -m 755 ${pathtocontainer}
```

Comment:
755 is to make sure that the actual uid to be started can read this folder, including configs and rootfs. Or the container will fail to start.